app/assets/javascripts/components/revisions/revision.jsx

```diff
@@ -9,17 +9,20 @@ const Revision = ({ revision, index, wikidataLabel, course, setSelectedIndex, la
   const ratingMobileClass = `${ratingClass} tablet-only`;
   const formattedTitle = CourseUtils.formattedArticleTitle({ title: revision.title, project: revision.wiki.project, language: revision.wiki.language }, course.home_wiki, wikidataLabel);
   const subtitle = wikidataLabel ? `(${CourseUtils.removeNamespace(revision.title)})` : '';
+  const isArticle = course.home_wiki.project !== 'wikidata';
```

Comment:
I don't think switching this behavior on the course home wiki is the best option here. The revisions in the Activity view may include ones from wikis other than the home wiki; a course that tracks both Wikipedia and Wikidata could have a mix of edits from either project.

Since these ratings are specific to Wikipedia (and are also irrelevant for other non-Wikipedia wikis in addition to Wikidata), I think checking whether the revision is for Wikipedia is probably the best option (ie, `isWikipedia = revision.wiki.project == 'wikipedia`).